editor/blocks/quote/index.js

```diff
@@ -0,0 +1,44 @@
+const Editable = wp.blocks.Editable;
+const { query, html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: html( 'cite,footer' )
+	},
+
+	edit( attributes, onChange ) {
+		const { value, citation } = attributes;
+
+		// Concat strings parsed by hpq from paragraph values into one string.
+		const combinedValue = value.join( '' );
+		return (
+			<blockquote>
+				<Editable
+					value={ combinedValue }
+					onChange={ ( newValue ) => onChange( { newValue } ) } />
+				<cite>
+					<Editable
+						value={ citation }
+						onChange={ ( newValue ) => onChange( { newValue } ) } />
```

Comment:
I think I understand that.  I am having trouble on figuring out where `onChange()` hooks up to in the code.  And where the `onChange` is being passed in.

I see :
```js
{ blocks.map( ( block, index ) =>
    <div key={ index }>
	{ wp.blocks.getBlockSettings( block.blockType ).edit( block.attributes ) }
    </div>
) }
```

I do not see where the onChange is passed in, in the code currently.  Or is there some magic going on somewhere?